### ISSUE TITLE
No support for passing Page Rotation as a property in Prawn::Document#generate

### ISSUE THREAD

**Issue by karthikselva** [prawnpdf/prawn#987]
- Prawn constructor accepts `:page_size` and `:page_layout` but it doesn't accept rotation angle of the PDF. 
- Even if I pass the `Prawn::Document.generate.generate(file,info: { Rotate: 270}}` It appends near meta info and but ideally it should append near mediabox 
## Sample PDF with rotation:

![image](https://cloud.githubusercontent.com/assets/202612/19092741/f389273e-8a3c-11e6-8fcf-5a7faed86840.png)
## Prawn Generated PDF:

![image](https://cloud.githubusercontent.com/assets/202612/19092797/3e779d16-8a3d-11e6-94f3-38dd42d8a41d.png)

**Comment by gettalong** [comment]
Yes, Prawn does not support an extra rotate key but that is okay as it is not needed. If you want to have a portrait document, use `orientation: :portrait` and if you want to have a landscape document use `orientation: :landscape`.

Note: Adding `:Rotate` to the `:info` key would just add it to the information dictionary but there no `:Rotate` key is supported.
